core-api/src/services/place.service.ts

```diff
@@ -70,4 +71,15 @@ export class PlaceService {
             throw new ServerError(e.message, ErrorCode.DATABASE_ERROR);
         }
     }
+
+    public static async findAccesses(id: number): Promise<SafeAccess[] | null> {
```

Comment:
This function is more important to the user then the place, so try to implement this type of function into UserService and UserController.

Reply:
Actually we won't need this if we set the relation on `place.entity.ts` with `eager: true`. This will tell Typeorm to populate the `accesses` attribute whenever we retrieve places from the repository (it will make automatic joins for us).
But in order to do this, @MateusPonciano should delete the `{ eager: true }` from lines 25 and 28 in `access.entity.ts` because cyclic eager loading relations are not supported.

Refer to https://github.com/typeorm/typeorm/blob/master/docs/relations.md#relation-options

Reply:
done what @gilesv suggested